.github/workflows/deploy_tre_reusable.yml

```diff
@@ -772,8 +782,8 @@ jobs:
           MGMT_STORAGE_ACCOUNT_NAME: ${{ secrets.MGMT_STORAGE_ACCOUNT_NAME }}
           SWAGGER_UI_CLIENT_ID: "${{ secrets.SWAGGER_UI_CLIENT_ID }}"
           USER_MANAGEMENT_ENABLED: ${{ vars.USER_MANAGEMENT_ENABLED }}
-          UI_SITE_NAME: "${{ vars.UI_SITE_NAME }}"
-          UI_FOOTER_TEXT: "${{ vars.UI_FOOTER_TEXT }}"
+          UI_SITE_NAME: "${{ inputs.UI_SITE_NAME || vars.UI_SITE_NAME }}"
```

Comment:
Where are the default values set? Looking above, are they `""`, rather than `"Azure TRE"` etc?